### PR TITLE
Add Related Website Set for stripe.com

### PR DESCRIPTION
**Pull request by erikb-stripe**
* Adds Related Website Set for stripe.com
* Well-known entries are in place for
  * https://stripe.com/.well-known/related-website-set.json (primary)
  * https://stripecdn.com/.well-known/related-website-set.json (service)
  * https://stripe.network/.well-known/related-website-set.json (service)
* Root of both service domains redirect to primary home page

### CODE REVIEW

**Review by aselya — APPROVED** [review]
LGTM, merging.

**Review by Parasos — COMMENTED** [review]
admin@stripe.com

File: related_website_sets.JSON [hunk 1]
```diff
@@ -487,6 +487,18 @@
           "https://rws2nvtvt.com": "Test 1",
```
Comment (Parasos) [inline related_website_sets.JSON:487]: "contact": "admin@stripe.com",
  "primary": "https://stripe.com",
  "serviceSites": [
     "https://stripecdn.com",
     "https://stripe.network"
  ],
  "rationaleBySite": {
     "https://stripecdn.com": "Serves static assets accelerating page loads and sandboxes select Javascript",
     "https://stripe.network": "Receives telemetry preventing

**Comment by Parasos** [comment]
"contact": "admin@stripe.com",
  "primary": "https://stripe.com",
  "serviceSites": [
     "https://stripecdn.com",
     "https://stripe.network"
  ],
  "rationaleBySite": {
     "https://stripecdn.com": "Serves static assets accelerating page loads and sandboxes select Javascript",
     "https://stripe.network": "Receives telemetry preventing

**Comment by Parasos** [comment]
66103d831d57290d2814bc9df8bafafea8e83bbd

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Comment by Parasos** [comment]
https://stripe.comc6fa8e0b804782e95cc2736fad7baeef2b0acc94

File: related_website_sets.JSON [hunk 2]
```diff
@@ -487,6 +487,18 @@
...
           "https://rws3nvtvt.com": "Test 2"
       }
+    },
+    {
```
Comment (Parasos) [inline related_website_sets.JSON:491]: related_website_sets.JSON"primary": "https://stripe.com"contact": "admin@stripe.com",
  "primary": "https://stripe.com",
  "serviceSites": [
     "https://stripecdn.com",
     "https://stripe.network"
  ],
  "rationaleBySite": {
     "https://stripecdn.com": "Serves static assets accelerating page loads and sandboxes select Javascript",
     "https://stripe.network": "Receives telemetry preventingadmin@stripe.com

File: related_website_sets.JSON [hunk 3]
```diff
@@ -487,6 +487,18 @@
...
+      "contact": "admin@stripe.com",
+      "primary": "https://stripe.com",
+      "serviceSites": [
+          "https://stripecdn.com",
```
Comment (Parasos) [inline related_website_sets.JSON:495]: "primary": "https://stripe.com

**Review by Parasos — COMMENTED** [review]
"primary": "https://stripe.com

File: related_website_sets.JSON [hunk 4]
```diff
@@ -487,6 +487,18 @@
           "https://rws2nvtvt.com": "Test 1",
           "https://rws3nvtvt.com": "Test 2"
```
Comment (Parasos) [inline related_website_sets.JSON:488]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Pull request merged**